{"name": "test_add_user[test_user_004-xx-1 348-883-4010-xx]", "status": "broken", "statusDetails": {"message": "AttributeError: 'Test_add_user' object has no attribute 'po'", "trace": "self = <test_add_user.Test_add_user object at 0x03A5C990>\n\n    @allure.step(title=\"点击新建联系人\")\n    @pytest.fixture\n    def add_user(self):\n>       self.po.click_find_user()\nE       AttributeError: 'Test_add_user' object has no attribute 'po'\n\nscripts\\test_add_user.py:29: AttributeError"}, "parameters": [{"name": "test_num", "value": "'test_user_004'"}, {"name": "name", "value": "'xx'"}, {"name": "phone", "value": "'1 348-883-4010'"}, {"name": "expect", "value": "'xx'"}], "start": 1600849017416, "stop": 1600849017416, "uuid": "2f78cb46-8392-4971-b1e1-6dffaf1d7052", "historyId": "200c96f572ab43af22aa18ac051ccdf2", "testCaseId": "79254fba797b53f4e834547c6c41889a", "fullName": "scripts.test_add_user.Test_add_user#test_add_user", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('add_user')"}, {"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_add_user"}, {"name": "subSuite", "value": "Test_add_user"}, {"name": "host", "value": "DESKTOP-BFU9FO1"}, {"name": "thread", "value": "8652-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_add_user"}]}